ui/widgets/sidebar/src/components/sidebar-new.tsx

```diff
@@ -0,0 +1,201 @@
+  onClickMenuItem: (appName: string, route: string) => void;
+  onClickExplore: () => void;
+  // viewport size
+  size?: string;
+  className?: string;
+  menuItem: React.ReactElement;
+
+  guestTitle: string;
+  guestSubtitle: string;
+  ctaText: string;
+  ctaButtonLabel: string;
+  footerLabel: string;
+  footerIcons: { name: IButtonProps['icon']; link: string }[];
+}
+
+const guestEthAddress = '0x00000000000000000000000000000';
+
+const titleText = 'text-sm font-bold';
+
+const subtitleText = 'text-xs text-grey5';
+
+const Sidebar: React.FC<ISidebarProps> = props => {
+  const {
+    userInstalledApps,
+    worldApps,
+    allMenuItems,
+    currentRoute,
+    activeApps,
+
+    guestTitle,
+    guestSubtitle,
+    ctaText,
+    ctaButtonLabel,
+    footerLabel,
+    footerIcons,
+
+    onSidebarClose,
+    onClickMenuItem,
+  } = props;
+
+  const [currentAppData, setCurrentAppData] = React.useState<IMenuItem | null>(null);
+  const [activeOption, setActiveOption] = React.useState<IMenuItem | null>(null);
+
+  React.useEffect(() => {
+    if (allMenuItems && currentRoute) {
+      const [, , , ...path] = currentRoute.split('/');
+
+      const activeApp = allMenuItems.find(menuItem => activeApps?.includes?.(menuItem.name));
+      if (activeApp && activeApp.index !== currentAppData?.index) {
+        setCurrentAppData(activeApp);
+      }
+      // set the subroute
+      if (path.length && currentRoute !== activeOption?.route) {
+        const currentOption = activeApp?.subRoutes?.find(
+          menuItem => menuItem.route === `/${path.join('/')}`,
+        );
+        if (currentOption && currentOption.index !== activeOption?.index) {
+          setActiveOption(currentOption);
+        }
+      }
+    }
+  }, [currentRoute, allMenuItems, currentAppData, activeOption, activeApps]);
+
+  const handleAppIconClick = (menuItem: IMenuItem, isMobile?: boolean) => {
+    if (menuItem.subRoutes && menuItem.subRoutes.length === 0) {
+      // if the current app has no subroutes, set as active and redirect to its route
+      setCurrentAppData(menuItem);
+      setActiveOption(null);
+      onClickMenuItem(menuItem.name, menuItem.route);
+      if (isMobile) {
+        onSidebarClose();
+      }
+    }
+  };
+
+  const handleOptionClick = (
+    menuItem: IMenuItem,
+    subrouteMenuItem: IMenuItem,
+    isMobile?: boolean,
+  ) => {
+    setCurrentAppData(menuItem);
+    setActiveOption(subrouteMenuItem);
+    onClickMenuItem(menuItem.name, subrouteMenuItem.route);
+    if (isMobile) {
+      onSidebarClose();
+    }
+  };
+
+  return (
+    <div
+      className={tw(
+        'max-w-[19.5rem] w-[19.5rem] max-h-[calc(100vh - 20px)] bg-white dark:bg-grey2 border-1 border-grey8 dark:border-none rounded-2xl',
+      )}
+    >
+      <div className={tw('flex flex-row p-4 border-b-1 border-grey8')}>
+        <div className={tw('w-fit h-fit mr-2')}>
+          <Avatar ethAddress={guestEthAddress} src={{ url: 'https://placebeard.it/360x360' }} />
```

Comment:
why do you use a placeholder like this? the avatar should have a default placeholder for the guest, but the src should be the actual avatar image of the profile? And why guestEthAddress? guest means the user is not connected, so it doesn't have any eth address